.github/workflows/errno.yml

```diff
@@ -0,0 +1,32 @@
+name: Error numbers
+on:
+  pull_request:
+    paths:
+      - 'lib/libc/minimal/include/errno.h'
+
+jobs:
+  check-errno:
+    runs-on: ubuntu-latest
+    container:
+      image: zephyrprojectrtos/ci:v0.17.1
+      options: '--entrypoint /bin/bash'
+    steps:
+      - name: Update PATH for west
+        run: |
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: west setup
+        run: |
+          west init -l . || true
+          west update
+
+      - name: Run errno.py
+        run: |
+          export ZEPHYR_BASE=${PWD}
+          ./scripts/ci/errno.py
```

Comment:
script should take required paths from input, e.g. `--zephyr-path`, `--zephyr-sdk-path`

Reply:
See my comment below, I am not convinced in this particular case.